templates/layouts/_base.html

```diff
@@ -73,32 +73,22 @@
 
 
 {% block head %}
-  {% if google_tag_manager_id and google_tag_manager_auth %}
-    <!--
-    Information such as events or variables can be passed to Google Tag Manager via the data layer,
-    and triggers can be set up in Google Tag Manager
-    -->
+  {% if google_tag_id %}
     <script nonce="{{ csp_nonce }}">
       dataLayer = {{ data_layer | tojson }}
     </script>
 
-    <!-- Google Tag Manager -->
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{ google_tag_id }}"></script>
```

Comment:
This script will always be loaded even if they haven't accepted the tracking cookies. The last script 
 for GTM dynamically added this to the Document, we should try to do the same.

Reply:
Updated now 👍